import { exitWithError, run } from "./run";

run().catch(exitWithError);
